palette/moreland/luminance_test.go

```diff
@@ -161,3 +161,11 @@ func BenchmarkLuminance_At(b *testing.B) {
 		})
 	}
 }
+
+func Test_luminance_Palette(t *testing.T) {
```

Comment:
```suggestion
func TestIssue798Kindlmann(t *testing.T) {
```

Reply:
Updated in 55662b36b4613f9934453412090318e259a40308.